### PR TITLE
10 exchanges API to get prices

### PR DESCRIPTION
**Pull request by rgoyal23**

### CODE REVIEW

File: coinbase.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import json
+import urllib2
```
Comment (kohlivarun5) [inline coinbase.py:2]: Using urllib2 instead of wget

File: coinbase.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  response = urllib2.urlopen(url, timeout = 5)
+  return json.load(response)
+
+def get_prices(download=True):
+  prices = {}
+  prices['bid'] = {}
+  prices['ask'] = {}
+  curs = ['btc', 'eth', 'ltc', 'bch']
+  for c in curs:
+    tmp = query(c.upper(),"sell")
```
Comment (kohlivarun5) [inline coinbase.py:15]: Why do we get `sell` prices when `bidding` ? Should it not be `ask` ?

File: coinbase.py (hunk 1 above)
Comment (rgoyal23) [reply coinbase.py:2]: cool

File: coinbase.py (hunk 2 above)
Comment (rgoyal23) [reply coinbase.py:15]: sell price is the price one can sell at, so passive bid price
in my dictionary, bid and ask are the passive prices

**Pull request merged**